include/nncase/compiler.h

```diff
@@ -81,6 +81,7 @@ class NNCASE_API compiler
     virtual ~compiler();
     virtual void import_tflite(std::span<const uint8_t> model, const import_options &options) = 0;
     virtual void import_onnx(std::span<const uint8_t> model, const import_options &options) = 0;
+    virtual void import_caffe(std::span<const uint8_t> model, const char *prototxt, const import_options &options) = 0;
```

Comment:
For API consistency, it's better to use `std::span<const uint8_t> prototxt`.

Reply:
Ok, will fix it later.